### ISSUE TITLE
Can't get aws pca to work

### ISSUE THREAD

**Issue by assafmanzur** [cert-manager/istio-csr#94]
https://github.com/cert-manager/istio-csr/issues/90#issuecomment-906957672
Workload keeps getting the `cluster.local` instead of my private domain

**Comment by SgtCoDFish** [comment]
I don't think this is enough data to debug, and this is very old in any case. Closing!
